functions

```diff
@@ -33,6 +33,7 @@ get() {
     closing-delimiter)      # shellcheck disable=SC2016
                             printf '```' ;;
     version-tag-prefix)     printf 'v' ;;
+    module-type)            printf 'commonjs' ;;
```

Comment:
Let's use the same order here as we do in the readme.

Reply:
:zap: